src/core/factcheck_plus/verdict.py

```
@@ -0,0 +1,141 @@
+        )
+
+
+def format_reason_line(internal: str, reason: str, supported_subclaim: str | None = None) -> str:
+    """Baut die verpflichtende Begründungszeile mit dem internen Grund.
+
+    Beispiele: ``"Teilweise bestätigt — belegter Teilclaim: …"`` bzw.
+    ``"Nicht überprüfbar — methodisch nicht herleitbar. …"`` (Spec §3/S5).
+
+    Args:
+        internal: Das interne Verdikt.
+        reason: Die Modellbegründung (1–2 Sätze).
```

Ruff (0.15.21)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)